app/experimenter/nimbus-ui/src/components/PageRequestReview/index.tsx

```diff
@@ -161,6 +162,46 @@ const PageRequestReview = ({
               </ChangeApprovalOperations>
             )}
 
+            <h5>Recommended actions before launch</h5>
+            <Table bordered data-testid="table-signoff" className="mb-4">
+              <tr data-testid="table-signoff-qa">
+                <td>
+                  <strong>QA Sign - Off</strong>
+                </td>
+                <td>
+                  {experiment.signoffRecommendations?.qaSignoff && (
```

Comment:
I think `qaSignoff` will always be `true` right? I guess it's good to leave the check in here anyway alongside the test, though if there was any possibility of this being `false` I can't help but think it'd be a little confusing without at least one `Recommended:` text guaranteed. As a user I'd just assume all of these were recommended🤷‍♀️

<img width="647" alt="image" src="https://user-images.githubusercontent.com/13018240/117186772-ccf65380-ada0-11eb-869d-9f9eb8d8c97c.png">

Reply:
Yeah but the assumption that it's always true is already hard coded in the backend, so if we hard code it here then it's done in two places.  And yeah I think what @AnaMedinac  was going for is that like, they're all kinda recommended, but if a risk mitigation question is true then they're a bit MORE recommended?  I think?

Reply:
Yeah I agree, just leave 'em in one place so if it ever changes we don't have to update the FE.

Also that sounds right as well, it's probably also fine since it says "recommended" above and not "required"